server/src/main/java/io/eiren/util/logging/LogManager.java

```diff
@@ -27,10 +27,11 @@ public static void initialize(File mainLogDir)
 		if (mainLogDir != null) {
 			if (!mainLogDir.exists())
 				mainLogDir.mkdirs();
-			File lastLogFile = new File(mainLogDir, "log_last.log");
-			if (lastLogFile.exists())
-				lastLogFile.delete();
-			FileHandler filehandler = new FileHandler(lastLogFile.getPath(), true);
+			for (File f : mainLogDir.listFiles()) {
+				if (f.getName().startsWith("log_last"))
+					f.delete();
+			}
+			FileHandler filehandler = new FileHandler("log_last_%g.log", 25 * 1000000, 2);
```

Comment:
`mainLogDir` should not be ignored for this file path. (This suggestion would also require an additional import for `import java.nio.file.Path;`)
```suggestion
			String lastLogPattern = Path.of(mainLogDir.getPath(), "log_last_%g.log").toString();
			FileHandler filehandler = new FileHandler(lastLogPattern, 25 * 1000000, 2);
```

Reply:
added the mainLogDir path!